include/lsst/afw/detection/Peak.h

```diff
@@ -194,7 +200,7 @@ class PeakTable : public afw::table::BaseTable {
     friend class afw::table::io::FitsWriter;
 
      // Return a writer object that knows how to save in FITS format.  See also FitsWriter.
-    virtual PTR(afw::table::io::FitsWriter) makeFitsWriter(fits::Fits * fitsfile, int flags) const;
+    std::shared_ptr<afw::table::io::FitsWriter> makeFitsWriter(fits::Fits * fitsfile, int flags) const override;
```

Comment:
@pschella's new guidelines for using `override` say to not use `virtual` as well.  I don't personally have a strong opinion.